package.json

```diff
@@ -4,7 +4,9 @@
   "description": "A protocol for state channel networks",
   "main": "lib/src/index.js",
   "files": [
-    "lib/*"
+    "lib/*",
+    "contracts/*",
+    "build/*"
```

Comment:
I don't think we want to include `build/*`, since that folder changes every time you deploy to any network.

I think it should be a  folder that is either:
A. not touched by any script
B. only touched by a script whose sole purpose is to prepare compiled contracts for publication

I'd suggest B, even running this script in the pre-publish script.

Reply:
I'll remove `build.*` from the `files` entry. It was more a short term solution until @kerzhner work with the deployer is done, but there's no real reason to include it now anyways.